
try {
    // code to be executed
} catch (error) {
    console.log(error);
}